packages/docs/src/stories/components/header.test.stories.ts

```diff
@@ -163,10 +186,10 @@ export const SampleA = {
             <!-- bottom-right-area start !-->
             <div class="-mr-4 flex items-center">
               <sd-navigation-item>
-                <div class="w-4 h-4 flex items-center"><sd-icon title="Unsere weiteren Auftritte" name="system/website" class="text-xl absolute -ml-[4px]"></sd-icon></div>
+                <div class="w-4 h-4 flex items-center"><sd-icon title="Unsere weiteren Auftritte" name="system/website" class="text-xl absolute -ml-[4px]" label="Other locations"></sd-icon></div>
```

Comment:
I believe we can remove the title-attrbute then, right?

Reply:
Not necessarly. The title provides visual indication for people that can see the icon but don't understand it. I would keep it. What do you think @mariohamann?

Reply:
Ah. Hm. Do we do this everywhere, where we `sd-icon` without text? I would like to follow a general rule here. Either everywhere or nowhere.

From my feeling `title` is a false promise – it's e. g. not working for Mobile Users and therefore fixes only something for some...

Reply:
Having a quick look on the codebase, I think we mostly don't use it, so to keep the consistency I will also remove it here.

> not working for Mobile Users and therefore fixes only something for some

This is true but can be said for other things aswell, we unfortunately aren't able to cover everything.